src/community/__stories__/blocks-story.tsx

```diff
@@ -44,55 +50,60 @@ export const BlockRow: StoryComponent<RowBlockArgs> = ({title, description, show
                             </StackingGroup>
                         ) : null
                     }
-                    aria-label="aria-label"
+                    aria-label={ariaLabel}
                 />
             </Box>
         </ResponsiveLayout>
     );
 };
 
-BlockRow.storyName = 'Row Block';
+BlockRow.storyName = 'RowBlock';
 BlockRow.args = {
     title: 'title',
     description: 'description',
     showStackingGroup: false,
+    ariaLabel: 'aria-label',
```

Comment:
I think it doesn't make sense to include a storybook control for ariaLabel, as the result of changing it isn't visual.